calendar/calendar.go

```diff
@@ -65,3 +65,38 @@ func RandomTimeInRange(startHour int, endHour int, loc *time.Location) time.Time
 	rangeStart := time.Date(year, month, date, startHour, 0, 0, 0, loc)
 	return rangeStart.Add(offsetDuration)
 }
+
+func CustzRandomTimeInRange(mtbf string, startHour, endHour int, loc *time.Location) time.Time {
```

Comment:
Would you mind using a different name? Not entirely sure what custz means.

Reply:
I'm a bit confused by the way this method is used.
If I set a mtbf of 1h I would expect my victim to be attacked roughly every hour, or 24 times during a single day.
However looking at the code, this will only schedule a single attack for that day.
Can you please clarify if that's the intention?